### ISSUE TITLE
NS_ERROR_FAILURE

### ISSUE THREAD

**Issue by MiD-AwE** [TiddlyWiki/TiddlyFox#28]
I just lost about two hours of documentation work due to this error. Even though it showed me this message it also displayed the "Saved wiki" bubble each time I clicked the save button. Please change the "Saved wiki" bubble to alert the user if the wiki in fact has no saved due to an error.

[Exception... "Component returned failure code: 0x80004005 (NS_ERROR_FAILURE) [nsILocalFile.create]"  nsresult: "0x80004005 (NS_ERROR_FAILURE)"  location: "JS frame :: chrome://tiddlyfox/content/overlay.js :: TiddlyFox.saveFile :: line 96"  data: no]

I am using version 5.1.7 recently upgraded from 5.0.8 Tiddlywiki with TiddlyFox 1.0alpha18.
The Tiddlywiki is save in My Documents on win8 x64.

**Comment by Jermolene** [comment]
Hi @MiD-AwE I'm sorry you lost your work, always very frustrating to hear of that happening.

Did the wiki filename include spaces or accented characters? I assume it was stored on a local drive?

**Comment by MiD-AwE** [comment]
Yes the name contains spaces. I used the name that was assigned after assigning all settings. So the name is a conjunction of the title and subtitle similar to "TiddlyWiki — a non-linear personal web notebook".

**Comment by Jermolene** [comment]
Hi @MiD-AwE OK, does the filename include the long dash that TiddlyWiki inserts in the page title? I think that may be the problem if so.

**Comment by MiD-AwE** [comment]
I do believe that is it. I will make that change first thing tomorrow.

**Comment by creinig** [comment]
FYI: I had the exact problem here (FF 37.0.1 on Windows 7). Renaming the wiki file to "wiki.html" solved the issue, as indicated by Jermolene.

I suggest changing the title of the default empty wiki to something more conservative. I'm sure other new users have that problem as well and many are discouraged enough by it to leave before looking at this ticket.

**Comment by Jermolene** [comment]
Thanks @creinig, I do intend to fix this problem but haven't had a chance to do any work on TiddlyFox for a while.

I've created a new ticket #29 that better describes the cause of the problem.
